src/Electrical/Digital/logic_vectors.jl

```diff
@@ -0,0 +1,47 @@
+import Base: size, axes, getindex
+
+const LogicOrNumber = Union{Logic, Number}
+
+struct StdULogicVector{N} <: AbstractArray{Logic, N}
+    logic::AbstractArray{Logic}
+    level::AbstractArray{Int}
```

Comment:
I notice that there's an abstract type here. I realize that MTK will butcher most type stability anyways and that this may not be a problem in this case, but have the implications of this been thought through?

Reply:
Now this is changed to `Array`